.github/workflows/companion-deploy.yml

```
@@ -3,7 +3,7 @@ name: Companion Deploy
 on:
   push:
     tags:
-      - '@uppy/companion@*'
+      - '@uppy/companion@**'
```

possibly... anyways this is probably not the solution
```suggestion
      - '@uppy/companion@*'
```